packages/@okta/vuepress-site/docs/reference/rate-limits/index.md

```diff
@@ -37,16 +35,14 @@ If an org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
 
 * [Concurrent rate limits](/docs/reference/rl-additional-limits/#concurrent-rate-limits): To protect the service for all customers, Okta enforces concurrent rate limits, which is a limit on the number of simultaneous transactions. Concurrent rate limits are distinct from the org-wide, per-minute API rate limits, which measure the total number of transactions per minute. Transactions are typically very short-lived. Even very large bulk loads rarely use more than 10 simultaneous transactions at a time.
 
-* [Client-based rate limits](/docs/reference/rl-clientbased/): Client-based rate limiting for the `/authorize` endpoint uses a combination of the Client ID, user's IP address, and Okta device identifier to provide granular isolation for requests made to the `/authorize` endpoint. This framework isolates rogue OAuth clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
+* [Client-based rate limits](/docs/reference/rl-clientbased/): Client-based rate limiting uses a combination of the client ID, user's IP address, and Okta device identifier (for the `/oauth2/v1/authorize` endpoint) or the IP address and device identifier (for the `/login/login.htm` endpoint) to provide granular isolation for requests made to the `/oauth2/v1/authorize` or `/login/login.htm` endpoint. This framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
```

Comment:
I find this sentence a bit hard to follow. Perhaps another bullet list, one level lower, could help?

Reply:
I reworded and shortened the sentence the best I could. Was going to create a sub-bullet list, but noticed that it would stand out in that particular section of the page.